File: Day 047/homework/hw3.py
# 3) დაწერეთ ფუნქცია, რომელიც არგუმენტად მიიღებს მომხმარებლის სახელს, გვარს და ასაკს. ფუნქციამ უნდა დააბრუნოს მომხმარებლის მონაცემები წინადადების სახით. (გამოიყენეთ f string-ი)





def data(name, surname, age):
    
    return (f"your name is {name},  your surname is {surname} and your age is {age}")

name = input ("Enter your name: ")
surname = input ("Enter your surname: ")
age = input ("Enter your age: ")

print(data(name, surname, age))

File: Day 105/homework/hw3.js
// 3) age ცვლადში შეინახეთ prompt-ის საშუალებით შემოტანილი მომხმარებლის ასაკი. გამოიყენეთ Ternary Operator, რომ დაბეჭდოთ:
// 'Adult' თუ მოხმარებლის ასაკი 18 წელზე მეტია, 'teen' თუ 13-დან 17-ის ჩათვლითაა, ხოლო 'child' ყველა სხვა შემთხვევაში.

const age = prompt("Enter your age: ")

let isAdult = age > 18 ? "Adult" :  age >= 13 && age <= 17 ? 'Teen' : age < 13 ? "Child" : NaN
console.log(isAdult)


